mobile/lib/services/preview_video_store.dart

```diff
@@ -529,16 +530,23 @@ class PreviewVideoStore {
     _logger.info("Getting playlist for $file");
     int? width, height, size;
     try {
-      final objectKey =
-          FileDataService.instance.previewIds?[file.uploadedFileID!]?.objectId;
-      final FileInfo? playlistCache = (objectKey == null)
-          ? null
-          : await cacheManager.getFileFromCache(_getCacheKey(objectKey));
-      final detailsCache = (objectKey == null)
-          ? null
-          : await cacheManager.getFileFromCache(
-              _getDetailsCacheKey(objectKey),
-            );
+      late final String objectKey;
+      final PreviewInfo? previewInfo =
+          FileDataService.instance.previewIds?[file.uploadedFileID!];
+      (String, String)? previewURLResult;
+      if (previewInfo == null) {
+        previewURLResult = await _getPreviewUrl(file);
```

Comment:
Can we do an additional check to only run this condition when file is not owned?

Reply:
I didn't see any particular reason for adding that check. Previously, if the file data information was missing, we were fetching and previewing it without caching anything as cacheID was null.

Let me know if I am missing some context.